syntax = "proto3";

package rpc_package;


service ClipService {
    rpc function_operate_clip (raw_rgb) returns (rgb_feature) {}
}


message raw_rgb {
    string my_rgb=1;
    int32 width=2;
    int32 height=3;
}

message rgb_feature {
    repeated float s_list=1;
}